int32 my_integer
geometry_msgs/Pose my_pose